test/test/fail_init.c

```diff
@@ -1,10 +1,25 @@
+/**
+ * @file test/test/fail_init.c
+ *
+ * @brief Test: expected failure, \ref test_config.test_init_fnc failure
+ *
+ * @test Tests that a failure in the test initialization function
+ * \ref test_config.test_init_fnc call causes a test failure.
```

Comment:
@ref?

Reply:
I'll change those to the more syntactically consistent _@ref_ tags.